Sources/SourceKitLSP/Swift/SyntaxHighlightingToken.swift

```diff
@@ -91,6 +92,7 @@ public struct SyntaxHighlightingToken: Hashable {
       switch self {
       case .namespace: return "namespace"
       case .type: return "type"
+      case .actor: return "class" // Map 
```

Comment:
Did you intend to write more for that comment?

Reply:
Oh, yes. I got distracted while writing it.